OSDOCS-4927: Release note for node port issue (#55214)
openshift/openshift-docs: microshift_release_notes/microshift-4-12-release-notes.adoc

* There is an issue with the way {product-title}'s CNI driver manages the firewall rules. If the firewall settings are modified after {product-title} is started then {product-title} must be restarted. The recommended workaround is to configure firewall rules before starting {product-title}.

> * There is an issue with new installs of {product-title} in which the `NodePort` service traffic might be stopped. To work around this issue, manually restart the `ovnkube-master` pod in the `openshift-ovn-kubernetes` namespace.

Reviewer: `installation` instead of `install`

> * There is an issue with new installs of {product-title} in which the `NodePort` service traffic might be stopped. To work around this issue, manually restart the `ovnkube-master` pod in the `openshift-ovn-kubernetes` namespace.

Reviewer: ```suggestion
* In the new installations of {product-title}, the `NodePort` service traffic might be stopped. To work around this issue, manually restart the `ovnkube-master` pod in the `openshift-ovn-kubernetes` namespace.
```
Just a suggestion, based on the my understanding, to keep the sentence minimal. Feel free to ignore if the suggestion doesn't technically sounds correct.
